File: Programming/C++/00/0015 Skocimis.cpp
#include<bits/stdc++.h>
using namespace std;
int number[10];

int main()
{
    // Input 3 integers
    for(int i=0;i<3;i++)
    {
        cin >> number[i];
    }

    // Output
    // Maximum turn is the longest distance - 1
    cout << std::max(number[1]-number[0], number[2]-number[1]) - 1 << endl;

    // End og the program
    return 0;
}

File: Programming/Python/00/0015 Skocimis.py
# Input 3 integers
number = [int(e) for e in input().split()]

# Find a difference between A,B and B,C
difference = [number[1]-number[0], number[2]-number[1]]

# Output
# Maximum turn is the longest distance - 1
print(max(difference) - 1)